### PR TITLE
Update Svenska.java

### PR DESCRIPTION
**Pull request by Padnezz**
Added the last empty translations. Some things to note:

- I added my name next to "SnickarN" in optionalCommentOnLanguage().

- In the first message in apiTimeOutException(), about a grandmother, in Sweden we have two different words for grandmother (one for fathers mother, and one for mothers mother), so I added them both.

- In apiTimeOutException() there is a google link with .de as a top-level domain, I changed that to .se

### CODE REVIEW

**Comment by MrMorkat** [comment]
Hey, first of all thank you for completing the translation ^^
Please fix remaining errors (Travis is complaining about a missing random object atm) and indentation. @omkelderman will look at it as soon as possible. :)

**Comment by MrMorkat** [comment]
Also, please consider Discord for further communication, you will also get a fancy translators role if you want :P

**Comment by omkelderman** [comment]
first glance: looks pretty gud, thanks 😄 

I'll take a final look and merge it when im home tonight o/

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Svenska.java [hunk 1]
```diff
@@ -182,23 +183,41 @@ public String setFormat() {
 		return "Syntaxen för att ställa en parameter är !set. Testa !help om du behöver mer hjälp.";
 	}
 	
+	StringShuffler doSomething = new StringShuffler(rnd);
+	
 	@Override
 	public String apiTimeoutException() {
-		return new Default().apiTimeoutException();
+		final String message = "osu! servrarna är jätte sakta just nu, så jag kan inte göra något för dig just nu.";
```
Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Svenska.java]: add a space as last character of this string (look at default.java to see what I mean)

**Pull request merged**